msi/GitProduct.wxs

```diff
@@ -266,6 +266,8 @@
   <Fragment>
     <SetProperty Id="ExecConfigureCoreAutoCrlf" Value='"[#GitExe]" config -f "[#GfwGitConfig]" core.autocrlf [LINEENDINGS]' Sequence="execute" Before="ExecConfigureCoreAutoCrlf" />
     <CustomAction Id="ExecConfigureCoreAutoCrlf" BinaryKey="WixCA" DllEntry="WixQuietExec$(var.SixtyFourBit)" Execute="deferred" Return="ignore" Impersonate="no" />
+    <SetProperty Id="ExecConfigureSslCAInfo" Value='"[#BashExe]" -c "git config -f /c/ProgramData/Git/config http.sslCAinfo \"/[#ca_bundle.crt]\""' Sequence="execute" Before="ExecConfigureSslCAInfo" />
```

Comment:
On Sat, 7 May 2016, Craig E. Shea wrote:

> > @@ -266,6 +266,8 @@
> >    <Fragment>
> >      <SetProperty Id="ExecConfigureCoreAutoCrlf" Value='"[#GitExe]" config -f "[#GfwGitConfig]" core.autocrlf [LINEENDINGS]' Sequence="execute" Before="ExecConfigureCoreAutoCrlf" />
> >      <CustomAction Id="ExecConfigureCoreAutoCrlf" BinaryKey="WixCA" DllEntry="WixQuietExec$(var.SixtyFourBit)" Execute="deferred" Return="ignore" Impersonate="no" />
> > +    <SetProperty Id="ExecConfigureSslCAInfo" Value='"[#BashExe]" -c "git config -f /c/ProgramData/Git/config http.sslCAinfo \"/[#ca_bundle.crt]\""' Sequence="execute" Before="ExecConfigureSslCAInfo" />
> 
> Yes, I fixed this for each `.gitconfig` option in the next two commits. I'll be squashing those fixes to the appropriate commits when I next clean up this branch. I don't actually use the environment variable at all (since, as you say, XP doesn't have a `%ProgramData%` environment variable)--but I assigned the `.gitconfig` file an ID in the installer so that when the ID is referenced, it is replaced with the installed path to the file. So, where ever it's installed, that's the file the command will act against.

That is an elegant solution!